### PR TITLE
PHPStan extension installer support

### PR DESCRIPTION
**Pull request by MartinMystikJonas**

### CODE REVIEW

**Comment by lulco** [comment]
Can you add info to readme like in https://github.com/phpstan/phpstan-nette?

```
If you also install [phpstan/extension-installer](https://github.com/phpstan/extension-installer) then you're all set!
Manual installation
```

**Comment by MartinMystikJonas** [comment]
Docs changed based on phpstan-nette

File: README.md [hunk 1]
```diff
@@ -21,23 +21,27 @@ The extension is based on Tomas Votruba's [blog series](https://tomasvotruba.com
...
 Add this line to your phpstan.neon:
 ```neon
 includes:
     - vendor/efabrica/phpstan-latte/rules.neon
 ```
 
+</details>
+
 It is recommended to use this extension with [phpstan-nette](https://github.com/phpstan/phpstan-nette) extension for better analysis of templates.
-Add also these lines to your phpstan.neon:
```
Comment (lulco) [inline README.md:35]: Do we want to remove this? User have to go to phpstan-nette and read its docs...

**Review by lulco — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (MartinMystikJonas) [reply README.md:35]: I think we should not repeat docs of another extensions.

**Pull request merged**